tests/fields.test.js

```diff
@@ -145,7 +145,7 @@ describe('dataObjectToArray', () => {
       f: 33,
     };
     const dataArray = dataObjectToArray(dataObject, fields);
-    expect(dataArray).toEqual(['', '33.333', '', '', 'a; b; c', '33', '', '']);
+    expect(dataArray).toEqual(['', '33.333', '', '', 'a;b;c', '33', '', '']);
```

Comment:
The delimiter downstream is just a semicolon, not a semicolon and a space. If we do indeed want it to be semicolon and a space, I need to discard this change, and update the delimiter downstream instead.